### PR TITLE
#5619 Add new Contact from Case Contacts tab

### PR DESCRIPTION
**Pull request by rdutu-vg**
**Description:**
Created test
Added locators
Created EditContactsSteps class with specific steps and methods
Added validations

Allure:
allure

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditContactsSteps.java [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+          webDriverHelpers.clickOnWebElementBySelector(CONTACT_CREATED_POPUP);
+          contactUUID = webDriverHelpers.getValueFromWebElement(UUID_INPUT);
+        });
+
+    Then(
+        "I verify that created contact from Case Contacts tab is correctly displayed",
+        () -> {
+          openContactFromResultsByUUID(contactUUID);
+          collectedContact = collectContactData();
+          SoftAssertions softly = new SoftAssertions();
```
Comment (BojteTamas) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditContactsSteps.java:111]: Please use the injection of the object

Comment (rdutu-vg) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditContactsSteps.java]: Done

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java [hunk 2]
```diff
@@ -21,5 +21,5 @@
 import org.openqa.selenium.By;
 
 public class EventParticipantsPage {
-  public static final By ADD_PARTICIPANT_BUTTON = By.cssSelector("div#eventParticipantAddPerson");
```
Comment (richardbartha) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java:24]: why did you delete this ?

Comment (rdutu-vg) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java:24]: removed unused variables

Comment (richardbartha) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java:24]: thats strange hopefully will not be deleted when merged into the dev branch :)

Comment (rdutu-vg) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java:24]: on my branch is unused, but if is used in current development, when I'll do the rebase we'll have it back again :D

**Review by richardbartha — APPROVED** [review]

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java (hunk 2 above)
Comment (rdutu-vg) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventParticipantsPage.java]: @richardbartha just rebased, and added back the locator
